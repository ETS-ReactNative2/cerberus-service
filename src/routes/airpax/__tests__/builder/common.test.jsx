import renderer from 'react-test-renderer';
import renderBlock from '../../TaskDetails/builder/helper/common';

describe('Commons Utils', () => {
  it('should render a jsx block from given', () => {
    const section = renderer.create(renderBlock('Reference', ['TR-172653'])).toJSON();
    expect(section).toMatchSnapshot();
  });
});
