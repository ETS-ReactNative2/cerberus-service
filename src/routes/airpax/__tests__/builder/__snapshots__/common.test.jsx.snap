// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Commons Utils should render a jsx block from given 1`] = `
<ul>
  <li
    className="govuk-grid-key font__light"
  >
    Reference
  </li>
  <li
    className="govuk-grid-value font__bold"
  >
    TR-172653
  </li>
</ul>
`;
